getting-started/chat-thread.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system each chat is divided into threads. Threads are standalone parts of a chat and can contain events. Multiple users (customers or agents) can participate in a chat and every user can have multiple chats at the same time.
+
+New threads within a chat are created on the server side, algorithm of this is discussed below in [Advanced](#advanced) section.
+
+![Chats and Threads](../images/chats.png "chats and threads")
+
+Events are portions of data which can be sent to a chat (e.g. in messages). See the next section of this guide for details (coming soon).
+
+### Getting chats
+
+If you log in to the Agent or Customer API you will receive one of the following in login response: chats_summary or last_chats_summary. Both of these objects contain some chat and thread IDs and these can be used to retrieve the chat history.
```

Comment:
Both of these objects contain some chat and thread IDs and these can be used to retrieve the chat history. ->  Both of these objects contain some chat and thread IDs - they can be used to retrieve the chat history.